packages/builder/.gitignore

```diff
@@ -5,4 +5,5 @@ package-lock.json
 release/
 dist/
 routify
-.routify/
\ No newline at end of file
+.routify/
+.rollup.cache
```

Comment:
How come this is needed?

Reply:
Rollup is now building the ts files, and it creates this cache folder with a lot of bundled files. We don't want this to be in the repo

Reply:
![image](https://github.com/user-attachments/assets/58674e9f-0eea-420d-acff-94c55eceab80)